server/routes/auth.js

```diff
@@ -0,0 +1,93 @@
+const express = require("express");
+const jwt = require("jsonwebtoken");
+
+const userController = require("../controllers/usersController");
+
+const router = express.Router();
+
+router.post("/login", validationMiddleware, async function (req, res, next) {
+    try {
+        const { email, password } = req.body;
+        // verify that there is user for email
+        const user = await userController.findOneWithemail(email);
+        if (!user) {
+            res.status(400).json({ errors: ["Email or password invalid"] });
```

Comment:
Let's make the error message as specific as possible - 'No user exists for that email' or something

Reply:
We could, but wouldn't want to give out the hackers easy way to find out which emails/usernames exists in out system. 

Thoughts? @sagar-13 @megangilpin @sina-jamshidi